Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+ADD generate.py /
+ENV TOKEN $TOKEN
+ENV BUCKET $BUCKET
+ENV GOOGLE_APPLICATION_CREDENTIALS $GOOGLE_APPLICATION_CREDENTIALS
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir git+git://github.com/suckatrash/infinitory.git@puppetdb_remote_queries
```

Comment:
Did you intend this to point to your fork?

Reply:
This is one of those chicken / egg things, I guess we just point it to puppetlabs / master

Reply:
It's a pip package - why not push it to artifactory or pypi and pull it from there in the Dockerfile?

Reply:
Reopening because I don't see an answer on the pip part - should we be building the python package and pushing it elsewhere and then consuming pypi/artifactor/etc here instead of hitting the repo directly?

Reply:
I'll push it to artifactory and update this.